src/store/index.js

```diff
@@ -12,5 +12,31 @@ export default new Vuex.Store({
   // tree: https://vuex.vuejs.org/guide/modules.html
   modules: {
     people
+  },
+  getters: {
+    // vuex.esm.js:95 Uncaught Error: [vuex] getters should be function but "getters.people" is {}.
```

Comment:
As the error suggests, Vuex getters should be a function so basically what you're trying to do below with `get` should be declared as a getter in the `people` module (store/people.js) instead of here in the root module.